cql2/standard/clause_4_terms_and_definitions.adoc

```diff
@@ -7,6 +7,11 @@ This document also uses terms defined in the OGC Standard for Modular specificat
 
 For the purposes of this document, the following additional terms and definitions apply.
 
+[[collation-def]]
+==== collation
+a set of rules that indicate how to compare and sort character string data
+the process of ordering units of textual information https://www.unicode.org/glossary/#collation[Glossary of Unicode Terms]
```

Comment:
The term “collation” does not seem to be used in the remainder of the document. Should it be removed from the terms & definitions?

Reply:
Meeting 2022-01-17: Use the term in the description and keep the definition.